Public/AdminPanel/js/modal-confirmation.js

```diff
@@ -34,23 +42,52 @@ let modalConfirmation = (function() {
                                         <p>${modalText}</p>
                                     </div>
                                     <div class="modal-footer">
-                                        <button type="submit" class="btn btn-${modalConfirmButtonType}">Confirm</button>
-                                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                                        <button type="submit" class="btn btn-${modalConfirmButtonType}">${confirmButtonText}</button>
```

Comment:
maybe namespace these variables just like the others? So `modalConfirmButtonText` and `modalDismissButtonText`?

Reply:
Never mind this one. It's just local variables and clear enough.